zetaclient/orchestrator/bootstrap.go

```diff
@@ -341,6 +342,7 @@ func syncObserverMap(
 				*params,
 				client,
 				tss,
+				time.Duration(cfg.RPCAlertLatency),
```

Comment:
Isn't `cfg.RPCAlertLatency` an int64? In this case we need to use seconds
```suggestion
				cfg.RPCAlertLatency * time.Second,
```

Reply:
It's multiplied by `time.Second` in the `base/observer.go`. Also, I unified it a bit more in commit https://github.com/zeta-chain/node/pull/2751/commits/05139aa9dd5c14cde3fd81fe4ec3f1fb1202dbe6